### ISSUE TITLE
Unable to get recod.csv, vgg16_weights_tf_dim_ordering_tf_kernels.h5 and vgg16_weights_tf_dim_ordering_tf_kernels.h5

### ISSUE THREAD

**Issue by zafrankhan** [AarohiSingla/Faster-R-CNN-on-Custom-Dataset#2]
Hi,
I have implemented Faster RCNN on thesame dataset of OpenImage as directed in video.but i am unable to get these three file.
1).   vgg16_weights_tf_dim_ordering_tf_kernels.h5
2).   record.csv
.can you please direct me in this regard. as i need to know the losses, classification accuracy and mean average precision of the model .
Thanks

**Comment by AarohiSingla** [comment]
You have to download the pretrained weights of VGG and put that weight file in your working directory. 
link: https://github.com/fchollet/deep-learning-models/releases/tag/v0.1
Download this weight file (vgg16_weights_tf_dim_ordering_tf_kernels.h5)
Then record.csv and other files will get created itself under model folder when you will execute your code.

**Comment by zafrankhan** [comment]
Thank you so much for the help.i am highly obliged for your kind response.Actually i have downloaded the weights of VGG16 as you mentioned.but still the record.csv file is not generated. what i have noticed is the though you have given output_weight_path,  record_path base_weight_path But you are saving the configuration only  and not the rest.please help me in this.
Thanks

**Comment by AarohiSingla** [comment]
You can download this code for reference. This is working fine on my PC.

**Comment by zafrankhan** [comment]
can you please amend the code and update in repository where each epoch is shown alongwith the results of that epoch during training. please
